### PR TITLE
Adjust link comment

### PR DESCRIPTION
**Pull request by yields**
Previously github would ignore the task code because it detects
that it was "autogenerated", this is an attempt to change the task
special comment so that github does not hide the diff.

### CODE REVIEW

File: pkg/runtime/javascript/javascript.go [hunk 1]
```diff
@@ -53,7 +53,7 @@ func (r Runtime) Slug(code []byte) (string, bool) {
 	for s.Scan() {
 		var line = strings.TrimSpace(s.Text())
 
-		if strings.HasPrefix(line, "// Generated by Airplane") {
+		if strings.HasPrefix(line, "// Linked to Airplane task:") {
```
Comment (joshma) [inline pkg/runtime/javascript/javascript.go:56]: Can you pull the prefix out to a constant, so it's easy to see that it's referenced somewhere else?

File: pkg/runtime/javascript/javascript.go [hunk 2]
```diff
@@ -72,5 +72,5 @@ func (r Runtime) Slug(code []byte) (string, bool) {
 
 // Comment implementation.
 func (r Runtime) Comment(t api.Task) string {
-	return fmt.Sprintf("// Generated by Airplane, do not edit:\n// %s", t.URL)
+	return fmt.Sprintf("// Linked to Airplane task:\n// %s", t.URL)
```
Comment (joshma) [inline pkg/runtime/javascript/javascript.go:75]: I liked the "do not edit" - maybe "Linked to Airplane task, do not edit:"?

**Review by joshma — APPROVED** [review]

File: pkg/runtime/javascript/javascript.go (hunk 2 above)
Comment (colinking) [reply pkg/runtime/javascript/javascript.go]: @yields @joshma wdyt of adding clarity on what shouldn't be edited?

```js
// https://app.airplane.dev/t/yolo [do not edit line]
```

Optionally, we could keep the first line as a clarifying comment, though our slug search logic can ignore it and just look for the first comment in a file that starts with `// https://app.airplane.dev/t/`.

```js
// The following line links this file to Airplane:
// https://app.airplane.dev/t/yolo [do not edit line]
```

Comment (joshma) [reply pkg/runtime/javascript/javascript.go]: I'm fine with either, what about one line?

    // Linked to https://app.airplane.dev/t/yolo [do not edit this line]

Comment (colinking) [reply pkg/runtime/javascript/javascript.go]: SGTM https://linear.app/airplane/issue/AIR-1230/adjust-link-comment

**Pull request merged**